models/index.js

```diff
@@ -1,5 +1,12 @@
 var mongoose = require("mongoose");
+const express = require('express');
```

Comment:
You don't need to require express and bodyParser in this file since you're not directly using them here. Those configurations can stay in the server.js file.